docs/conf.py

```diff
@@ -178,11 +176,11 @@
             'o_6.wgfa',
         ]
     )
-    with open(os.path.join(FIASCO_HOME, 'fiascorc'), 'w') as f:
+    with open(fiasco_home / 'fiascorc', 'w') as f:
         c = configparser.ConfigParser()
         c.add_section('database')
-        c.set('database', 'ascii_dbase_root', ascii_dbase_root)
-        c.set('database', 'hdf5_dbase_root', hdf5_dbase_root)
+        c.set('database', 'ascii_dbase_root', str(ascii_dbase_root))
+        c.set('database', 'hdf5_dbase_root', str(hdf5_dbase_root))
```

Comment:
This isn't really Pythonic, but using `str` here is because `ConfigParser.set()` doesn't work with `Path` instances.